_posts/blog/2016-08-21-gopherjs-1.7-1-release.markdown

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title:  "GopherJS 1.7-1 is released"
+date:   2016-08-21
+author: Dmitri Shuralyov
+---
+
+Hello gophers,
+
+Today, we are happy to make two announcements for GopherJS. First, GopherJS now [officially supports Go 1.7](https://github.com/gopherjs/gopherjs/issues/475), which is the current stable release of Go that came out recently. Second, GopherJS has its first named version attached to it, and we're calling it GopherJS 1.7-1. This makes it possible to have a release history tracking major changes and additions between versions.
```

Comment:
I would remove the link and bold the text:

> Today, we are happy to make two announcements for GopherJS. First, **GopherJS now officially supports Go 1.7**, which is the current stable release of Go that came out recently.

The link doesn't give me a whole lot / just distracts me.